kahuna/public/js/search/index.js

```diff
@@ -88,11 +92,25 @@ search.config(['$stateProvider', '$urlMatcherFactoryProvider',
                 description: 'Toggle metadata panel',
                 callback: panels.metadataPanel.toggleHidden
             });
+
+            keyboardShortcut.bindTo($scope).add({
+                combo: shortcutKeys.get('showCollections'),
+                description: 'Toggle collections',
+                callback: () => {
+                    const toggled = !storage.getJs('showCollectionsPanel');
+                    storage.setJs('showCollectionsPanel', toggled);
+                    $window.alert('Now refresh to start using collections');
```

Comment:
They might be on a search they don't want to lose. As this is hyper-temporary, I don't really mind asking.